### ISSUE TITLE
sed: can't read /etc/php/7.4/fpm/pool.d/www.conf: No such file or directory

### ISSUE THREAD

**Issue by brooksvb** [laravel/homestead#1387]
## Versions
  - Vagrant: 2.2.7
  - Provider: Virtualbox 6.1.4
  - Homestead: 8.0 (Composer package 10.6)

## Host operating system
Windows 10

## Homestead.yaml
```yaml
ip: 192.168.10.20
name: metools.test
hostname: metools.test
version: 8.0.0
memory: 2048
cpus: 1
provider: virtualbox
folders:
    -
        map: ./
        to: /home/vagrant/code
sites:
    -
        map: metools.test
        to: /home/vagrant/code/public
        type: "symfony4"
        php: "7.2"
databases:
    - metools
backup: true
```

## Vagrant destroy & up output
Vagrant up output: https://gist.github.com/brooksvb/f14196cf791f019cba1da8888d93be1f
There is no Vagrant destroy output because it errors on trying to make a database backup on the destroy trigger, because the database did not get created during provision. To test different changes I completely deleted the `.vagrant` directory, removed the VM from Virtualbox, and ran `vagrant box remove laravel/homestead`.

## Expected behavior
The virtual machine should provision and run correctly.

## Actual behavior
The provisioning of the virtual machine errors because php7.4 is not installed: `sed: can't read /etc/php/7.4/fpm/pool.d/www.conf: No such file or directory`

## Steps to reproduce
1. Run `vagrant up`

**Comment by svpernova09** [comment]
Change your folder mapping. You shouldn’t map ‘./‘ or ‘.’.

May need to manually destroy the vm in the provider time get around the db.

https://laravel.com/docs/7.x/homestead

> You should never mount . (the current directory) when using Homestead. This causes Vagrant to not map the current folder to /vagrant and will break optional features and cause unexpected results while provisioning.

--
- Joe Ferguson
JoeFerguson.me
osmihelp.org

**Comment by brooksvb** [comment]
Sorry, I always forget I can't do that. I changed the folder mapping to a full path, but I still got the same result:
https://gist.github.com/brooksvb/ab05808f223735f50aea7e81c3534160

**Comment by svpernova09** [comment]
You're also on a really old base box: `Box Version: 8.0.0` Make sure you're Homestead repo is up to date. Homestead is trying to write to PHP 7.4 files but they don't exist on your base box because your box is older than when we started supporting 7.4. You need to pull the most recent base box.

Also: You're running Virtualbox 6.1, which is the first I've seen it, so I'm betting there are issues there we may not know about yet. If other weird issues crop up I would try downgrading to 6.0.x and see if the issue remains.

**Comment by brooksvb** [comment]
Issue resolved by updating to `version: 9.3.0` in `Homestead.yaml`
